### DISCUSSION TITLE
CUDA out of memory

### DISCUSSION THREAD

**Discussion by Vi-cs** [zetavg/LLaMA-LoRA-Tuner#21] [Q&A]
Hi,

I regularly get this error when running the app, after a couple (4/5) inferences via the GUI:

`CUDA out of memory. Tried to allocate 502.00 MiB (GPU 0; 22.19 GiB total capacity; 19.98 GiB already allocated; 194.50 MiB free; 20.10 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
`

My current workaround is to kill the process to free some memory.

Did I miss a configuration step or something that prevent out of memory?

**Comment by zetavg** [comment]
Hi! You can try to use the `--load_8bit` flag when running `python app.py`, which will enable `load_in_8bit` for models and save some VRAM.

**Comment by Vi-cs** [comment]
thanks!!
